target/linux/generic/pending-5.10/485-mtd-spi-nor-add-xmc-xm25qh128c.patch

```diff
@@ -0,0 +1,17 @@
+From: Kyle Liu <lyq123233@gmail.com>
+Subject: kernel/mtd: add support for XMC XM25QH128C
+Date: Tues, 25 Jan 2022 09:22:41 +0800
+
+Signed-off-by: Kyle Liu <lyq123233@gmail.com>
+
+--- a/drivers/mtd/spi-nor/xmc.c
++++ b/drivers/mtd/spi-nor/xmc.c
+@@ -14,6 +14,8 @@ static const struct flash_info xmc_parts
+ 			    SECT_4K | SPI_NOR_DUAL_READ | SPI_NOR_QUAD_READ) },
+ 	{ "XM25QH128A", INFO(0x207018, 0, 64 * 1024, 256,
+ 			     SECT_4K | SPI_NOR_DUAL_READ | SPI_NOR_QUAD_READ) },
++	{ "XM25QH128C", INFO(0x204018, 0, 64 * 1024, 256,
++                             SECT_4K | SPI_NOR_DUAL_READ | SPI_NOR_QUAD_READ) },
```

Comment:
Some whitespace issue here?

Reply:
Yes, I will change it

Reply:
While at it, fix your `Signed-off-by: lyq1996 <lyq123233@gmail.com>`  as in https://openwrt.org/submitting-patches#sign_your_work

> using your real name (no pseudonyms or anonymous contributions.) "